### PR TITLE
Adds Perl version detection

### LINKED ISSUE #2203: Add Perl, which is installed by default in Ubuntu 18.04
**Description**  

`perl 5.26` is included by default as a system tool in Ubuntu 18.04. It should be listed as part of the tools.
Just running `perl -v` in an Ubuntu runner will show it's working.

**Feature?**:  

**Virtual environments affected**  
- [ ] Ubuntu 16.04
- [x] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Expected behavior**  

If it's available, it should be listed alongside the others.

**Actual behavior**  

It's not

**Repro steps**  

Add these steps to an action running `ubuntu-latest`

```
      - name: Comprueba perl
        run: perl --version
```

### PR DESCRIPTION
**Pull request by JJ**
# New tool

Perl is currently installed in all major Linux distributions, but it was not detected. This adds detection, guided by #2204. It fixes #2203 

#### Related issue:

#2203 #2204 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -49,6 +49,12 @@ function Get-NodeVersion {
     return "Node $nodeVersion"
 }
 
+function Get-PerlVersion {
+    $result = Get-CommandResult "perl --version | perl -ne 'print /(\d\.\d+\.\d+)/'"
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:53]: nitpick: `perl -e 'print substr($^V, 1)'`

Comment (JJ) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:53]: So true...

Comment (JJ) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:53]: Um, having trouble with the $ inside a PS command...

Comment (JJ) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1]: I've used that, but it didn't work for some reason (probably related to the $ thingie) inside `Get-CommandResult` I can go back to that version if calling it is mandatory.

**Comment by JJ** [comment]
FWIW, perl was eliminated from the default configuration in Ubuntu 20.04 IARC. If you want me to add something to it so that it's installed, it would be my pleasure.

**Comment by al-cheb** [comment]
> FWIW, perl was eliminated from the default configuration in Ubuntu 20.04 IARC. If you want me to add something to it so that it's installed, it would be my pleasure.

Thank you. Looks like we  preinstall perl on Ubuntu 20.04:
image

**Review by maxim-lobanov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Comment by maxim-lobanov** [comment]
@JJ , thank you for contribution!

**Comment by JJ** [comment]
My pleasure! Thanks for doing this so fast.

**Pull request merged**